entries/c/caisse-epargne.fr.json

```diff
@@ -2,11 +2,14 @@
   "Caisse D'Epargne": {
     "domain": "caisse-epargne.fr",
     "url": "https://www.caisse-epargne.fr/",
-    "contact": {
-      "facebook": "Caisse.Epargne",
-      "twitter": "CaissEpargneSAV",
-      "language": "fr"
-    },
+    "tfa": [
+      "sms",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Sécur’Pass"
+    ],
+    "documentation": "https://www.caisse-epargne.fr/comptes-cartes/dsp2/#anchor-authentification-forte",
```

Comment:
What's described sounds more like authorization for specific user actions than authentication at the login phase.
We distinguish between the two in the section about authorization in [Contributing.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization).
Please confirm if this is the case or if 2FA is also prompted on login.

Reply:
I confirm sadly that french banks only use 2FA for authorization and not for each logiin with account/password